src/cmd-build

```diff
@@ -8,7 +8,7 @@ dn=$(dirname "$0")
 print_help() {
     cat 1>&2 <<'EOF'
 Usage: coreos-assembler build --help
-       coreos-assembler build [--force] [--skip-prune]
+       coreos-assembler build [--force] [--skip-prune] [--timestamp-only]
```

Comment:
This feels like it'd be cleaner as `prune --timestamp-only` since we already have the code there right?

Reply:
I struggled with this; I used `build` because it seemed to make a bit more sense to use it with that verb.  Using it with `prune` feels odd.

Maybe we need another verb that can operate on the metadata itself?

That idea aside, I'll move the timestamping to `prune` in the next commit.

Reply:
New verb `touch`?  Or `bump`?  Or something...

Reply:
Yeah, putting it under `prune` feels odd to me. I'd agree just making it a separate verb, though that seems overkill as well. How about just make all actions (e.g. `build`, `prune`, `buildextend*`) call out to a script that just bumps it? So basically the timestamp reflects whenever it was  last edited, which I think is the intent we're trying to convey here.

Anyway, I know you *just* changed it so we can do this as a follow-up :)

Reply:
![jlebonnnnnn](https://user-images.githubusercontent.com/10236257/48723438-a48c3600-ebf4-11e8-83d4-8a18fc40e3fa.jpg)